main.go

```diff
@@ -640,12 +709,12 @@ func main() {
 					},
 				},
 			},
-			// {
-			// 	Name:    "console",
-			// 	Aliases: []string{"con", "c"},
-			// 	Usage:   "Federate into the AWS console",
-			// 	Action:  fedConsole,
-			// },
+			{
+				Name:    "console",
+				Aliases: []string{"con", "c"},
+				Usage:   "Federate into the AWS console",
+				Action:  fedConsole,
+			},
```

Comment:
Was this uncommented intentionally?  As far as I know this is still unsupported.